test/REQUIRE

```diff
@@ -1 +1,2 @@
-SpecialFunctions
\ No newline at end of file
+RecipesBase
```

Comment:
Shouldn't this go to `REQUIRE` (as opposed to `test/REQUIRE`)? Or am I missing something?

Reply:
I doubt you are missing anything; I am just not very familiar with pre-1.0 dependency management